_data/languages.yml

```diff
@@ -1,6 +1,6 @@
 en:
   label: English
-  icon: 🇺🇸
+  icon: 🇬🇧
```

Comment:
![brit](https://media.giphy.com/media/Qws1zLvQDG7kI/giphy.gif)

Reply:
I will never approve this

Reply:
![a](https://media1.tenor.com/images/c92e712171f1e1079464aba7f9dd8be7/tenor.gif?itemid=9375594)